h/services/annotation_json_presentation.py

```diff
@@ -13,15 +13,15 @@
 
 
 class AnnotationJSONPresentationService(object):
-    def __init__(self, session, user, group_svc, links_svc, flag_svc, moderation_svc, has_permission):
+    def __init__(self, session, user, group_svc, links_svc, flag_svc, flag_count_svc, moderation_svc, has_permission):
```

Comment:
Not your fault, but this is getting increasingly cumbersome to manage, and the tests as well. In the past we didn't really want to pass a request to a service, but maybe we should think about an exception for this. Or come up with a different solution.